src/css/index.css

```diff
@@ -1,5 +1,8 @@
 @import './brand-colors.css';
-@import './light-theme-colors.css';
-@import './dark-theme-colors.css';
+@import './brand-colors-deprecated.css';
 @import './typography.css';
 @import './shadow.css';
+@import './light-theme-colors.css';
```

Comment:
reordered since some [deprecated] theme colors do reference shadow

Reply:
I think these can be reverted as there is no more deprecated classnames